### PR TITLE
Show new UI in Web3Signer package

### PR DESCRIPTION
**Pull request by dsimog01**
Shown new UI in web3signer

### CODE REVIEW

File: dappnode_package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "mgarciate <mgarciate@gmail.com> (https://github.com/mgarciate)",
     "pablomendezroyo <mendez4a@gmail.com> (https://github.com/pablomendezroyo)",
     "eduadiez <eduadiez@gmail.com> (https://github.com/eduadiez)"
```
Comment (pablomendezroyo) [inline dappnode_package.json:15]: missing comma

File: dappnode_package.json [hunk 2]
```diff
@@ -33,5 +34,13 @@
   },
   "warnings": {
     "onRemove": "Make sure your web3signer does not have this client selected or you will stop validating! (Packages > web3signer > config > client)"
-  }
+  },
+  "globalEnvs": [
+    {
+      "envs": ["CONSENSUS_CLIENT_PRATER", "EXECUTION_CLIENT_PRATER"],
```
Comment (pablomendezroyo) [inline dappnode_package.json:40]: It should be `CONSENSUS_CLIENT_MAINNET` instead of `CONSENSUS_CLIENT_PRATER`

File: dappnode_package.json [hunk 3]
```diff
@@ -33,5 +34,13 @@
...
+      "services": ["ui"]
+    },
+    { "envs": ["CONSENSUS_CLIENT_PRATER"], "services": ["web3signer"] }
```
Comment (pablomendezroyo) [inline dappnode_package.json:43]: same as above

File: ui/.env [hunk 4]
```diff
@@ -0,0 +1,4 @@
+NETWORK=mainnet
+WEB3SIGNER_API_URL=http://web3signer.web3signer.dappnode:9000/
+CONSENSUS_CLIENT=$_DAPPNODE_GLOBAL_CONSENSUS_CLIENT
```
Comment (pablomendezroyo) [inline ui/.env:3]: missing `MAINNET`

**Review by pablomendezroyo — CHANGES_REQUESTED** [review]

**Comment by dsimog01** [comment]
After fix, the package has been built and can be installed with this IPFS hash: /ipfs/QmS2u1Azk2hFuSJoN3sEDL3oo339jz3MtbgiaVnhVhnkeC

**Review by pablomendezroyo — APPROVED** [review]

**Pull request merged**